### DISCUSSION TITLE
当表格内容出现换行时，单元格内容会出现错乱。导致下游任务识别出现问题

### DISCUSSION THREAD

**Discussion by Ethenlalala** [PaddlePaddle/PaddleOCR#13766] [Q&A]
**原始图片**
![image](https://github.com/user-attachments/assets/c833fdcc-8b6a-4a76-acd7-2a7aff05c150)
**识别结果**
![image](https://github.com/user-attachments/assets/c3d7de17-a638-4e20-8ece-e03a08f40533)
问题如标题描述，有什么好的办法可以将每个单元格的内容进行拼接还原吗？

**Comment by SWHL** [comment]
搞一个识别表格的模型先来做cell的识别，之后再将同个cell的文字拼接起来
